experimental/manifests/base/argo/workflow-controller-configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+data:
+  config: |
+    {
+    executorImage: argoproj/argoexec:v2.2.0,
+    artifactRepository:
+    {
+        s3: {
+            bucket: mlpipeline,
```

Comment:
Maybe rename `mlpipeline` to `kubeflow-pipelines` or `kf-pipelines` here and in other places?

Reply:
we can do a renaming in a dedicated PR. for now it might be better to align with the existing one.